client/signup/config/flows-pure.js

```diff
@@ -129,7 +130,9 @@ export function generateFlows( {
 				'site-topic-with-preview',
 				'site-title-with-preview',
 				'site-style-with-preview',
-				'domains-with-preview',
+				abtest( 'skippableDomainStep' ) === 'skippable'
```

Comment:
Will this `abtest()` method run when the flow is first setup, or right before the domains step renders? The latter is preferable, since we should be setting AB's as close to the testable event as possible to achieve significance in a shorter period of time. For instance, if we set it at the beginning of Signup, a good deal of users assigned to the test will never make it to the domains step. I believe that's the thinking behind `preloadABTestVariationsForStep()` and `getABTestFilteredFlow()` in [signup/config/flows.js](https://github.com/Automattic/wp-calypso/pull/29494/files#diff-5efc50beea9eac4cabe6d62cb7c92e52R265)

Reply:
Thanks for the tip! When testing I see that flows are generated even if you just navigate to `/`! Which makes me wonder why `signup/config/flows.js` would be included in that bundle in the first place 🤔 

Something like `getABTestFilteredFlow()` is better, but even that calls `abtest()` earlier than we probably want. I liked the idea of running the test using flow config but this will probably always assign users to test groups early than we'd like.

It might be better to call `abtest()` inside the step component since that's the first time we actually _need_ to know which test group the user is in.